test/units/ansible_test/test_validate_modules.py

```diff
@@ -1,12 +1,13 @@
 """Tests for validate-modules regexes."""
 from __future__ import annotations
 
-import mock
 import pathlib
 import sys
 
 import pytest
 
+from units.compat import mock
```

Comment:
Since this is a controller-only test it doesn't need to use the compat layer.

```suggestion
from unittest import mock
```